postman/Get Clicks.postman_collection.json

```diff
@@ -0,0 +1,577 @@
+							"protocol": "https",
+							"host": [
+								"api",
+								"aweber",
+								"com"
+							],
+							"path": [
+								"1.0",
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}",
+								"campaigns",
+								"b{{broadcastId}}"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Messages in Broadcast",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "0826d611-cf14-4a5c-a835-ac43669d9fca",
+								"exec": [
+									"pm.test(\"Successfully got the messages int the broadcast\", function(){",
+									"    ",
+									"})"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{messages_collection_link}}",
+							"host": [
+								"{{messages_collection_link}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Get Followup Email Clicks",
+			"item": [
+				{
+					"name": "Get Number of Followups on List",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "1bac9b8a-de19-4264-9c19-bdc1ea154a5c",
+								"exec": [
+									"pm.test(\"Successfully got number of followups in list\", function() {",
+									"    pm.expect(parseInt(responseBody)).to.be.a('number');",
+									"    var followup_entry = Math.floor((Math.random() * parseInt(responseBody) ))",
+									"    pm.globals.set(\"followup_entry\",followup_entry )",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_URL}}/accounts/{{accountId}}/lists/{{listId}}/campaigns?campaign_type=f&ws.op=find&ws.show=total_size",
+							"host": [
+								"{{base_URL}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}",
+								"campaigns"
+							],
+							"query": [
+								{
+									"key": "campaign_type",
+									"value": "f"
+								},
+								{
+									"key": "ws.op",
+									"value": "find"
+								},
+								{
+									"key": "ws.show",
+									"value": "total_size"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Random Followup",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "d2b1b30c-3aec-48cc-9f35-c1ab0683ee45",
+								"exec": [
+									"pm.test(\"Successfully got a followup email\", function() {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.entries[pm.globals.get(\"followup_entry\")].id).to.be.a('number');",
```

Comment:
This can fail if you have more followups than fit in a page. This should pick from the ones returned in this page, not the total.